elasticdl_client/api.py

```diff
@@ -62,14 +65,15 @@ def init_zoo(args):
  --extra-index-url={{ EXTRA_PYPI_INDEX }}
 
 {% if CLUSTER_SPEC_NAME  %}\
-COPY ./{{ CLUSTER_SPEC_NAME }} /cluster_spec/{{ CLUSTER_SPEC_NAME }}\
+COPY ./{{ CLUSTER_SPEC_NAME }} {{CLUSTER_SPEC_PATH}}/{{ CLUSTER_SPEC_NAME }}\
 {% endif %}
 """
     template = Template(tmpl_str)
     docker_file_content = template.render(
         BASE_IMAGE=args.base_image,
         EXTRA_PYPI_INDEX=args.extra_pypi_index,
         CLUSTER_SPEC_NAME=cluster_spec_name,
+        CLUSTER_SPEC_PATH=ClusterSpecConfig.CLUSTER_SPEC_DIR,
```

Comment:
CLUSTER_SPEC_PATH -> CLUSTER_SPEC_FOLDER_PATH

Reply:
CLUSTER_SPEC_DIR ?

Reply:
> CLUSTER_SPEC_DIR ?

This name is better